src/it/issue-mscmpub-50/pom.xml

```diff
@@ -83,7 +104,7 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-project-info-reports-plugin</artifactId>
-        <version>@projectInfoReportsPluginVersion@</version>
+        <version>3.4.1</version> <!-- Updated version -->
```

Comment:
we don't want to change to a fixed version from the variable; maybe update the variable definition instead